bdk_chain/src/lib.rs

```diff
@@ -55,4 +57,5 @@ pub mod collections {
     pub type HashSet<K> = hashbrown::HashSet<K>;
     pub type HashMap<K, V> = hashbrown::HashMap<K, V>;
     pub use alloc::collections::*;
+    pub use hashbrown::hash_map::Entry;
```

Comment:
```suggestion
    pub use hashbrown::hash_map;
```

I think this would make more sense, then the above two lines can be removed and everything will be consistent.

Reply:
What exactly are you referring as inconsistent?

Reply:
I meant that you can remove the two lines above without resulting in inconsistency.

Reply:
Same this as here https://github.com/LLFourn/bdk_core_staging/pull/118?notification_referrer_id=NT_kwDOAi2U5bM1MTkyOTA5MTQ4OjM2NTQxNjY5&notifications_query=reason%3Aparticipating#discussion_r1060037236